Source/JavaScriptCore/runtime/IntlDisplayNames.cpp

```diff
@@ -137,6 +138,10 @@ void IntlDisplayNames::initializeDisplayNames(JSGlobalObject* globalObject, JSVa
         throwTypeError(globalObject, scope, "failed to initialize DisplayNames"_s);
         return;
     }
+#else
+        throwTypeError(globalObject, scope, "failed to initialize Intl.DisplayNames since feature is not supported by the ICU version"_s);
+        return;
```

Comment:
This looks like it’s indented too many spaces

Reply:
Oops, you're right.